ComAmazonawsDynamodb/codegen-patches/java/dafny-4.2.0.patch

```diff
@@ -129,20 +129,9 @@ index 171a36d3..3964b7cb 100644
    public static DynamoDbClient DynamoDB_20120810(IDynamoDBClient dafnyValue) {
      return ((Shim) dafnyValue).impl();
    }
-@@ -8054,17 +8174,9 @@ public class ToNative {
-       return (DynamoDbException) dafnyValue.dtor_obj();
+@@ -8055,6 +8175,8 @@ public class ToNative {
      } else if (dafnyValue.dtor_obj() instanceof Exception) {
        return (RuntimeException) dafnyValue.dtor_obj();
--    } else if (dafnyValue.dtor_message().is_Some()) {
```

Comment:
I don't know what this is about

Reply:
Changed in https://github.com/smithy-lang/smithy-dafny/pull/470,
this seems like the right change

Reply:
https://github.com/smithy-lang/smithy-dafny/pull/470